packages/fluent-theme/src/components/theme/Theme.module.css

```diff
@@ -623,3 +623,102 @@
   padding: var(--webchat-spacingVerticalSNudge) var(--webchat-spacingHorizontalM);
   transition: opacity var(--webchat-durationNormal) var(--webchat-curveDecelerateMid);
 }
+
+/* Feedback Form */
+:global(.webchat-fluent).theme :global(.webchat__feedback-form__body1) {
+  font-family: var(--webchat-fontFamilyBase);
+  color: var(--webchat-colorGray200);
+  font-size: var(--webchat-fontSizeBase300);
+  font-weight: var(--webchat-fontWeightRegular);
+  line-height: var(--webchat-lineHeightBase300);
+}
+
+:global(.webchat-fluent).theme :global(.webchat__feedback-form__caption1) {
+  font-family: var(--webchat-fontFamilyBase);
+  color: var(--webchat-colorNeutralForeground3);
+  font-size: var(--webchat-fontSizeBase100);
+  font-weight: var(--webchat-fontWeightRegular);
+  line-height: var(--webchat-lineHeightBase200);
+}
+
+:global(.webchat-fluent).theme :global(.webchat__feedback-form__button__submit) {
+  background-color: var(--webchat-colorBrandForeground1);
+  border: 1px solid var(--webchat-colorBrandForeground1);
+  border-radius: var(--webchat-borderRadiusMedium);
+  color: var(--webchat-colorNeutralBackground1);
+  cursor: pointer;
+  font-family: var(--webchat-fontFamilyBase);
+  font-size: var(--webchat-fontSizeBase300);
+  height: 24px;
+  padding: 0 var(--webchat-spacingHorizontalS);
+
+  &:hover {
+    background-color: var(--webchat-colorBrandForeground2Hover);
+    border: 1px solid var(--webchat-colorBrandForeground2Hover);
+  }
+
+  &:active {
+    background-color: var(--webchat-colorBrandForeground2Pressed);
+    border: 1px solid var(--webchat-colorBrandForeground2Pressed);
+  }
+}
```

Comment:
This and bellow button is missing focus Fluent styles:

```css
&:focus-visible {
    outline: var(--webchat-strokeWidthThick) solid var(--webchat-colorStrokeFocus2);
    outline-offset: calc(var(--webchat-strokeWidthThin) * -1);
}
```
Please check if the offset is needed.

Reply:
Please check the designs for the stroke thickness for the small buttons as well.